doc/api/api_changes_3.3/behaviour.rst

```diff
@@ -155,3 +155,12 @@ support for it will be dropped in a future Matplotlib release.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Previously, keyword arguments were silently ignored when no positional
 arguments were given.
+
+
+`.get_minorticklabels` and `.get_majorticklabels` now returns plain list
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Previously, `.get_minorticklabels` and `.get_majorticklabels` returns
```

Comment:
```suggestion
Previously, `.Axis.get_minorticklabels` and `.Axis.get_majorticklabels` returns
```
As the doc build log says:
~~~
/home/circleci/project/doc/api/api_changes_3.3/behaviour.rst:160: WARNING: more than one target found for cross-reference 'get_minorticklabels': matplotlib.axis.Axis.get_minorticklabels, matplotlib.axis.XAxis.get_minorticklabels, matplotlib.axis.YAxis.get_minorticklabels
~~~
i.e. Sphinx does not know where to link. One option, suggested here, is to qualify that with the class you want to reference.